src/swarm/node/model/IChannelsNodeInfo.d

```diff
@@ -174,13 +174,15 @@ version ( UnitTest )
             Params:
                 actions = list of record action types to track stats for
                 requests = list of request types to track stats for
+                neo_requests = list of neo request types to track stats for
                 channels = list of channels to track stats for
 
         ***********************************************************************/
 
-        this ( istring[] actions, istring[] requests, istring[] channels )
+        this ( istring[] actions, istring[] requests, istring[] neo_requests,
+            istring[] channels )
```

Comment:
Is this a breaking change?

Reply:
I think so. Well spotted. I'll have a look at it.

Reply:
Ah no. It's just a class used internally within swarm for unittesting. Of course, strictly speaking, code outside of swarm _could_ use this, but that would be very weird and is not intended.